src/wp-includes/functions.php

```diff
@@ -4310,7 +4471,8 @@ function _doing_it_wrong( $function, $message, $version ) {
 			} else {
 				$version = sprintf( '(This message was added in version %s.)', $version );
 			}
-			$message .= sprintf( ' Please see <a href="%s">Debugging in ClassicPress</a> for more information.',
+			$message .= sprintf(
+				' Please see <a href="%s">Debugging in ClassicPress</a> for more information.',
```

Comment:
same as above

Reply:
I agree, while we are pointing at upstream links the description should match.